next.dynamic.mjs

```diff
@@ -9,6 +9,8 @@ import { serialize } from 'next-mdx-remote/serialize';
 import * as nextLocales from './next.locales.mjs';
 import * as nextConstants from './next.constants.mjs';
 import * as nextHelpers from './next.helpers.mjs';
+import rehypeAutolinkHeadings from 'rehype-autolink-headings/lib';
+import rehypeSlug from 'rehype-slug';
```

Comment:
What is this plugin and why do we need it?

Reply:
well ```rehypeSlug```  add ids to the headings, ```rehypeAutolinkHeadings``` doesn't work as expected

Reply:
Wdym doesn't work as expected? It's a well known plugin. Maybe you're using it incorrectly? Please read the docs ;)